adm/_make_and_install_crontab.sh

```diff
@@ -19,10 +19,9 @@ echo_ok
 
 echo -n "- Installing crontab file..."
 echo_running
-deploycron_file "${MODULE_RUNTIME_HOME}/tmp/config_auto/crontab"
+deploycron_file "${MODULE_RUNTIME_HOME}/tmp/config_auto/crontab" > /dev/null 2>&1
 if test $? -eq 0; then
     echo_ok
 else
-    echo_nok
-    exit 1
+    echo_warning "Can't install crontab file (normal if you are in continuous integration with docker)"
```

Comment:
it would be interesting to add a similar test to `mfxxx.status`:

- if the crontab command does not exist => warning
- if the crontab command exists AND if the crontab is empty => nok (with a !=0 return code)